src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js

```diff
@@ -7,14 +7,14 @@ const IntegrationsGrid = ({ obj }) => {
     <HoneycombGrid>
       <div className="container">
         {obj.map((item,index) => {
-          const status = item.status === "InProgress" ? true : false;
+          const status = item.frontmatter.status === "InProgress" ? true : false;
```

Comment:
Filter for `published: true` in the graphql then this check won't be needed

Reply:
We need this check to display both InProduction and InProgress Integrations

Reply:
Didn't get you 🤔

Reply:
We are using this check to separate both the InProgress and InProduction Integration as we have to display both on the page and style them according to that. 

For example, if I just used the Filter you mentioned, it will query only those integration which is InProduction, not the InProgress stage.  So, we won't be able to display those integrations which are InProgress state.

Reply:
Aahh...ok.
Thanks for the clarification :)